doc/sphinx/mgmt_preview_quickstart.rst

```diff
@@ -0,0 +1,151 @@
+Quickstart Tutorial - Resource Management (Preview Libraries)
+===============================================================
+
+We are excited to announce that a new set of management libraries are now in Public Preview.
+Those packages share a number of new features such as Azure Identity support,
+HTTP pipeline, error-handling.,etc, and they also follow the new Azure SDK guidelines which
+create easy-to-use APIs that are idiomatic, compatible, and dependable.
+
+You can find the details of those new libraries `here <https://azure.github.io/azure-sdk/releases/latest/#python>`__
+
+In this basic quickstart guide, we will walk you through how to
+authenticate to Azure using the preview libraries and start interacting with
+Azure resources. There are several possible approaches to
+authentication. This document illustrates the most common scenario
+
+Prerequisites
+-------------
+
+| You will need the following values to authenticate to Azure 
+- **Client ID** 
+- **Client Secret** 
+- **Tenant ID** 
+- **Subscription ID**
+
+These values can be obtained from the portal, here's the instructions:
+
+Get Subscription ID
+^^^^^^^^^^^^^^^^^^^
+
+1. Login into your azure account
+2. Select Subscriptions in the left sidebar
+3. Select whichever subscription is needed
+4. Click on overview
+5. Copy the Subscription ID
+
+Get Client ID
+^^^^^^^^^^^^^
+
+1. Login into your azure account
+2. Select azure active directory in the left sidebar
+3. Click "App Registrations"
+4. Select the application which you have created
+5. Copy "Application (client) ID"
+
+Get Client Secret
+^^^^^^^^^^^^^^^^^
+
+1. Login into your azure account
+2. Select azure active directory in the left sidebar
+3. Click "App Registrations" in the left sidebar
+4. Click "Certificates & Secrets"
+5. Click "+ New client secret"
+6. Type description and click "Add"
+7. Copy and store the key value. You won’t be able to retrieve it after
+   you leave this page.
+
+Get Tenant ID
```

Comment:
I'd move it before secret as this is the same set of instructions as Client ID so people don't need to navigate between tabs back and forth